samples/cpp/cloning_gui.cpp

```diff
@@ -453,7 +453,7 @@ int main()
 
     for(;;)
     {
-        char key = (char) waitKey(0);
+        int key = waitKey(0) & 0xFF;
 
         if(key == 'd' && flag3 == 0)
```

Comment:
I mean that the `int key = waitKey(0) & 0xFF;` is always non-negative, while `char` constant (`'*'`) can be negative when the `char` type is signed and the symbol ASCII code is more than 127 - do you agree?
But in reality people will unlikely use regional-specific symbols is such case, so the issue is minor.